### PR TITLE
Fix the reference to the 'Configuring the Spatial Awareness Mesh Observer' page.

### PR DESCRIPTION
**Pull request by wiwei**
This particular entry in the TOC wasn't hooked up even though the underlying page was already created.

### CODE REVIEW

**Comment by davidkline-ms** [comment]
That's odd. I recall writing that article...

File: Documentation/toc.yml [hunk 1]
```diff
@@ -85,8 +85,6 @@
         href: README_Solver.md     
     - name: Spatial Awareness
       href: SpatialAwareness/SpatialAwarenessGettingStarted.md
-      items:
-      - name: Configuring the Spatial Awareness Mesh Observer
```
Comment (davidkline-ms) [inline Documentation/toc.yml]: looks like the issue is actually a missing href line.

please add:
href: SpatialAwareness/ConfiguringSpatialAwarenessMeshObserver.md

Comment (wiwei) [reply Documentation/toc.yml]: Done.

**Pull request merged**